### DISCUSSION TITLE
Sidebar to check notes without leaving workflow

### DISCUSSION THREAD

**Discussion by fithurriague** [nvim-neorg/neorg#111] [Ideas]
It would be really handy to have a command (with it's own shortcut) to accede a nerdtree-like menu of ur principal workspace notes. So while coding you can **quickly** change something in your notes and continue working without having to worry about the notes being somewhere in your open buffers or having to fuzzyfind them.

**Comment by mrossinek** [comment]
I think this is rather out-of-scope for neorg itself. A proper implementation of this would require a manual support for a **multitude** of file-explorer plugins. Instead, I suggest that this be handled by the user configuration on a per-usecase basis. I believe that most file-explorer plugins support a way of opening a specific directory in a buffer (be that a split or a floating window). Since you configure the locations of your Neorg workspaces yourself, too, you can easily configure a function/command to open a file-explorer at that location.

Here is an example for a Telescope mapping to fuzzy find over all files in a workspace:
```vim
nnoremap <silent> <leader>of <cmd>lua require'telescope.builtin'.find_files(require'telescope.themes'.get_dropdown( { previewer = false, prompt_title = "~ neorg ~", cwd = "~/neorg", find_command = { "rg", "--ignore", "--hidden", "--files", "--no-ignore-vcs", "--glob", "!.git" } } ) )<CR>
```
